<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogGrayscale</class>
 <widget class="QDialog" name="DialogGrayscale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Grayscale</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>81</width>
     <height>61</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="rLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>R:</string>
   </property>
  </widget>
  <widget class="QLabel" name="gLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>G:</string>
   </property>
  </widget>
  <widget class="QLabel" name="cLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>B:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="rDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.299000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="gDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.587000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="bDoubleSpinBox">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.114000000000000</double>
   </property>
  </widget>
  <widget class="QCheckBox" name="averageCheckBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>70</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Computes the unweighted average based on the RGB coefficients.</string>
   </property>
   <property name="text">
    <string>Average</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogGrayscale</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogGrayscale</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
